samples/boards/nrf52/mesh/onoff_level_app/README.rst

```diff
@@ -0,0 +1,101 @@
+.. _bluetooth-mesh-onoff-sample:
+
+Bluetooth: Mesh OnOff & Level Model
+###########################
+
+Overview
+********
+
+This is a simple application demonstrating a Bluetooth mesh node.
+
+Root element has Generic OnOff Server, Generic OnOff Client, Generic Level Server, Generic Level Client models.
+
+Prior to provisioning, an unprovisioned beacon is broadcast that contains
+a unique UUID. It is obtained from the device address set by Nordic in the
+FICR. Each button controls the state of its
+corresponding LED and does not initiate any mesh activity.
+
+Association of Models with h/w (in case of nRF52840-PDK board)
```

Comment:
Recommend spelling out ``hardware`` and rewrite this section as:

    Associations of Models with hardware
    ************************************

    For the nRF52840-PDK board, these are the model associations:

    * LED1 is associated with generic OnOff Server
    * Button1 and Button2 are associated with generic OnOff Client: 

      * [Button1 : ON]
      * [Button2: OFF]
    * LED3 is associated with generic Level Server:

      * [if (Level < 50) LED3: OFF else LED3: ON]
    * Button3 and Button4 are associated with generic Level Client: 

      * [Button3: publishes Level = 25]
      * [Button4: publishes Level = 100]

Note the blank lines and indentation is important.